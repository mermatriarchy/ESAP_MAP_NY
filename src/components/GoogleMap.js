import React from 'react';

export default function GoogleMap() {
  return (
    <div>
      Google Map goes here.
    </div>
  );
}
