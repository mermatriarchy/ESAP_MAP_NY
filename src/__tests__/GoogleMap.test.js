import React from 'react';
import renderer from 'react-test-renderer';

import GoogleMap from '../components/GoogleMap';

it('correctly renders About component', () => {  
    const GoogleMapComponent = renderer.create(<GoogleMap />).toJSON();
    expect(GoogleMapComponent).toMatchSnapshot();
});