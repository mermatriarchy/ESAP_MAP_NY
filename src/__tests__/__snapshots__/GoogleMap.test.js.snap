// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correctly renders About component 1`] = `
<div>
  Google Map goes here.
</div>
`;
